profiles/schema.py

```diff
@@ -109,11 +109,12 @@ def _create_nested(model, profile, data):
                 raise
 
 
-def _update_nested(model, profile, data):
+def _update_nested(model, profile, data, field_cb=lambda *args: None):
```

Comment:
Ha, that's pretty much what I had, but then I went to this direction :laughing: 

One difference though, as a first line within the function I had `field_callback = field_callback if field_callback else lambda *args: None`, in order to avoid an `if` clause inside a `for` loop. How does that feel?

Reply:
I would just use if inside the if. It's the most easy to read. The loop is at most tens of items(?). So no need to optimise. In my opinion.